system_files/deck/shared/usr/lib/sysctl.d/65-hpet.conf

```diff
@@ -0,0 +1 @@
+dev.hpet.max-user-freq=3072
```

Comment:
This is not needed when the file exists in the desktop image, as the deck image is built from the desktop image.
If we needed the number to be different for the deck image then this file would be needed.